sources/config_reader.c

```diff
@@ -1724,7 +1744,79 @@ static int od_config_reader_route(od_config_reader_t *reader, char *db_name,
 		return NOT_OK_RESPONSE;
 
 	/* unreach */
-	return od_config_reader_rule_settings(reader, rule, extentions, NULL);
+	return od_config_reader_rule_settings(reader, rule, extentions, NULL,
+					      NULL);
+}
+
+static int od_config_reader_group(od_config_reader_t *reader, char *db_name,
```

Comment:
default way to write and use od_config_reader_something it is to give reader and pointer to struct/variable where you should read

look at od_config_reader_string or od_config_reader_number64 or od_config_reader_keyword

lets give in arguments od_group_t* ?

Reply:
thanks, fixed